src/ja/common/config.py

```diff
@@ -0,0 +1,13 @@
+from ja.common.message import Serializable
+
+
+class Config(Serializable):
```

Comment:
I want to ask because obviously I don't understand what these classes are for.
I thought config classes are like default values for job, mount paths, etc. In the end, we'd transfer the Job itself, and not the config. So why is the config serializable at all? Is it just for reading it from a YAML file?

Reply:
Yes.

Reply:
Disregard my last comment.
We're going to transfer the Config which has a nested Job.
The Config will also contain metadata.

Reply:
I am not sure about this. What is the metadata?

Reply:
I just realized my comment was again incorrect because I misused the word metadata.

We are going to transfer Message objects.
We are not going to transfer Config objects.

Config objects contain all the necessary information to encode a CLI command.
They can contain data like the verbosity level which does not have a direct influence on Command effects.
It would also be where we would put options that are client-side-only (like user credentials).

Message objects are Serializable because we are transferring them as YAML.
Config objects are Serializable because we need to store them as YAML files on disk.

Metadata is stored in the Message object.
As of right now we are not planning to add relevant amounts of metadata but this is where metadata would be added.